packages/plugins/docs-ui/src/lib/components/links/document-attach-dialog.component.html

```
@@ -0,0 +1,42 @@
+<nb-card class="docs-dialog docs-attach-dialog">
+	<nb-card-header>{{ 'DOCS.LINKS.ATTACH_DIALOG_TITLE' | translate }}</nb-card-header>
+
+	<nb-card-body [nbSpinner]="loading" nbSpinnerStatus="primary">
+		<input
+			id="docs-attach-search"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)